Documentation/improve documentation (#127)
bmw-software-engineering/lobster: documentation/user-manual.md

duplicate maintenance and tedious manual processes. LOBSTER aims to
simplify this. The basic setup is as follows:

1. Configure LOBSTER via the tracing policy. Usually called `lobster.conf`. More on that [here](https://github.com/bmw-software-engineering/lobster/blob/main/documentation/config_files.md).

> 2. Annotate traces onto your code, your tests and/or your requirements. Please notice each one of the supported languages has their own way to recieve the tracing tags.

Reviewer: ```suggestion
2. Annotate traces onto your code, your tests and/or your requirements. Please notice each one of the supported languages has their own way to annotate tracing tags.
```

Author: done

> 3. Extract the requirement traces with the corresponding lobster conversion tools (e.g. `lobster-cpp`). This writes *.lobster file or files into a "common unified interchange format".

Reviewer: ```suggestion
3. Extract the requirement traces with the corresponding lobster conversion tools (e.g. `lobster-cpp`). This converts those traces into a *.lobster file (a "common interchange format").
```

Author: done

4. Build report by calling `lobster-report`
5. Render your report by calling the core losbter tools. Usually a local HTML report is desired via `lobster-html-report` or even `lobster-ci-report` for your CI.

The basic idea is that you have a number of artefacts that you wish to
relate to each other; stored in different "databases". Sometimes this
is very literal (e.g. with proprietary requirements tools like
codeBeamer), but often we just mean "somewhere in git".

> #### NOTE:

Author: done

* [lobster_gtest](manual-lobster_gtest.md): for tracing tags in
  (executed) googletests.

> > **Note:** CPP tool needs a clang-tidy file. Dont forget to generate it to be able to use this convertion tool.

Author: done
